### ISSUE TITLE
Unable to deploy: TypeError: Cannot read properties of null (reading 'getUsers')

### ISSUE THREAD

**Issue by deni64k** [tryretool/retool-onpremise#178]
Hello everyone!

I am using this command to deploy a self-hosted Retool

```
bash -c "$(curl -fsSL https://raw.githubusercontent.com/tryretool/retool-onpremise/ssop/deploy-retool)" < <(echo "SSOP_myguid")
```

 but `api` process crashes with this error
```
retool-onpremise-api-1               | {"message":"Rechecking license status...","level":"info","timestamp":"2023-05-09T00:42:19.589Z"}
retool-onpremise-api-1               | Encountered error
retool-onpremise-api-1               | TypeError: Cannot read properties of null (reading 'getUsers')
retool-onpremise-api-1               |     at checkLicenseStatus (/snapshot/retool_development/backend/transpiled/server/modules/licensing.js)
retool-onpremise-api-1               |     at async startOnPremServer (/snapshot/retool_development/backend/transpiled/server/main.js)
retool-onpremise-api-1               |     at async main (/snapshot/retool_development/backend/transpiled/server/main.js)
retool-onpremise-api-1               | Exiting...
retool-onpremise-api-1 exited with code 0
```

I'm using Google Compute Engine, `n2-standard-4`, and a 60 GB disk.

**Comment by deni64k** [comment]
I tried to rollback from `latest` (which 1.117) to `1.116.10` in `Dockerfile`, and now it works.

**Comment by BenjamynBaker** [comment]
Hello! This issue should be resolved in 2.117.3 - release notes [here](https://updates.retool.com/en?category=self-hosted%20release%20notes)
